src/Evolution/BoundaryCorrectionTags.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "NumericalAlgorithms/SpatialDiscretization/OptionTags.hpp"
+
+namespace evolution {
+namespace OptionTags {
+/// The boundary correction used for coupling together neighboring cells or
+/// applying boundary conditions.
+///
+/// In the finite volume/difference and discontinuous Galerkin literature this
+/// is ofter referred to as the "numerical flux". We avoid that nomenclature
+/// because in the discontinuous Galerkin and finite volume case it is not the
+/// flux that is modified, but the integrand of the boundary integral.
```

Comment:
I'm sure you've documented elsewhere the details of what terms appear in the DG calculation -- could you point to that documentation here, like 'e.g. the term [blah] in [nth equation] from the documentation for [OtherDgUtility]'?

(any changes to this documentation block, same to below)

Reply:
Currently spectre really doesn't document how to do DG, actually. This is something I discussed with @fmahebert and I told him I was planning on adding a long Dev Guide (or user tutorial?) on how we do DG. Maybe the thing to do is to create an issue where I can track everything that will need to point to the dev guide. Does that sound reasonable?

Reply:
That sounds good, I'm happy with an issue to keep track of the pieces that should link into that future documentation.

Reply:
Okay, created #2567 to keep track of these